exchanges/bybit/bybit_wrapper.go

```diff
@@ -943,48 +937,32 @@ func getOrderTypeString(oType order.Type) string {
 	}
 }
 
-// ModifyOrder will allow of changing orderbook placement and limit to
-// market conversion
+// ModifyOrder will allow of changing orderbook placement and limit to market conversion
```

Comment:
🐒 I think "orderbook placement" is a complex way of saying "changing the price" ?
```suggestion
// ModifyOrder modifies an existing order, including price and order type
```
or just
```suggestion
// ModifyOrder modifies an existing order
```

Reply:
This will change across all exchanges; think we should defer this for a standardisation pass. The second option is good. 👍

Reply:
Erm, yikes. Agreed.